### ISSUE TITLE
Missing Exception class

### ISSUE THREAD

**Issue by gerrywastaken** [m4i/rubocop-git#21]
```
./rubocop-git-0.1.1/lib/rubocop/git/style_guide.rb:75:in `target_ruby_version': uninitialized constant RuboCop::Git::StyleGuide::ValidationError (NameError)
Did you mean?  RuboCop::ValidationError
    from ./rubocop-git-0.1.1/lib/rubocop/git/style_guide.rb:36:in `parse_source'
```

It seems that ValidationError class isn't defined anywhere. You can get this issue to appear by simply not specifying a `TargetRubyVersion` in your .rubocop file.

**Comment by tansaku** [comment]
I get this too ...

**Comment by r00takaspin** [comment]
Same error with ruby-2.3.1

**Comment by m4i** [comment]
Thank you for your reporting. 
This bug was probably fixed in #26 and 0.1.2.
If the problem still occurs, please reopen.
